### ISSUE TITLE
Errors when using MultiQC: list passed instead of path

### ISSUE THREAD

**Issue by lbeltrame** [chapmanb/bcbio-nextgen#1344]
Master as of today:

```python
Traceback (most recent call last):
  File "/storage/rozo/bcbio/anaconda/bin/bcbio_nextgen.py", line 226, in <module>
    main(**kwargs)
  File "/storage/rozo/bcbio/anaconda/bin/bcbio_nextgen.py", line 43, in main
    run_main(**kwargs)
  File "/storage/rozo/bcbio/anaconda/lib/python2.7/site-packages/bcbio/pipeline/main.py", line 39, in run_main
    fc_dir, run_info_yaml)
  File "/storage/rozo/bcbio/anaconda/lib/python2.7/site-packages/bcbio/pipeline/main.py", line 82, in _run_toplevel
    for xs in pipeline(config, run_info_yaml, parallel, dirs, samples):
  File "/storage/rozo/bcbio/anaconda/lib/python2.7/site-packages/bcbio/pipeline/main.py", line 184, in variant2pipeline
    samples = qcsummary.generate_parallel(samples, run_parallel)
  File "/storage/rozo/bcbio/anaconda/lib/python2.7/site-packages/bcbio/pipeline/qcsummary.py", line 48, in generate_parallel
    samples = run_parallel("multiqc_summary", [samples])
  File "/storage/rozo/bcbio/anaconda/lib/python2.7/site-packages/bcbio/distributed/multi.py", line 28, in run_parallel
    return run_multicore(fn, items, config, parallel=parallel)
  File "/storage/rozo/bcbio/anaconda/lib/python2.7/site-packages/bcbio/distributed/multi.py", line 86, in run_multicore
    for data in joblib.Parallel(parallel["num_jobs"])(joblib.delayed(fn)(x) for x in items):
  File "/storage/rozo/bcbio/anaconda/lib/python2.7/site-packages/joblib/parallel.py", line 804, in __call__
    while self.dispatch_one_batch(iterator):
  File "/storage/rozo/bcbio/anaconda/lib/python2.7/site-packages/joblib/parallel.py", line 662, in dispatch_one_batch
    self._dispatch(tasks)
  File "/storage/rozo/bcbio/anaconda/lib/python2.7/site-packages/joblib/parallel.py", line 570, in _dispatch
    job = ImmediateComputeBatch(batch)
  File "/storage/rozo/bcbio/anaconda/lib/python2.7/site-packages/joblib/parallel.py", line 183, in __init__
    self.results = batch()
  File "/storage/rozo/bcbio/anaconda/lib/python2.7/site-packages/joblib/parallel.py", line 72, in __call__
    return [func(*args, **kwargs) for func, args, kwargs in self.items]
  File "/storage/rozo/bcbio/anaconda/lib/python2.7/site-packages/bcbio/utils.py", line 51, in wrapper
    return apply(f, *args, **kwargs)
  File "/storage/rozo/bcbio/anaconda/lib/python2.7/site-packages/bcbio/distributed/multitasks.py", line 143, in multiqc_summary
    return qcsummary.multiqc_summary(*args)
  File "/storage/rozo/bcbio/anaconda/lib/python2.7/site-packages/bcbio/pipeline/qcsummary.py", line 920, in multiqc_summary
    folders.append(os.path.dirname(pfiles))
  File "/storage/rozo/bcbio/anaconda/lib/python2.7/posixpath.py", line 122, in dirname
    i = p.rfind('/') + 1
AttributeError: 'list' object has no attribute 'rfind'
```

`pfiles` is a list, and not a string.

**Comment by chapmanb** [comment]
Luca;
Sorry about the issues. We're revamping the multiQC framework and also porting it to work with the common workflow language and it doesn't look like we handled qsignature correctly. I pushed fixes which I believe will resolve this if you update.

@lpantano -- do we have a minimal qsignature/MultiQC project somewhere we could test on? I don't think we're feeding qsignature into multiQC right now and would like to get that in the reports as well if it's supported.
